EIPS/eip-5252.md

```diff
@@ -0,0 +1,237 @@
+
+<figcaption align = "center"><b>Fig 1 - Stakers vs Enforcers</b></figcaption>
+
+**Stakers**:  Stakers are people who vote to enforcers' proposals and get dividend for staked tokens
+
+**Enforcers**: Enforcers are people who takes risk on managing protocol and contributes to the protocol by making a proposal and change to it.
+
+#### Contracts
+
+**`Influencer`**: **`Influencer`** contract stores influence configurations and measures the contribution of a user from his activities done in registered **`Account Bound Token(ABT)`** contract. The contract puts lock on **`Account Bound Token(ABT)`** until proposal is finalized.
+
+
+**`Governor`**: **`Governor`** contract is compatible with current governor contract in Openzeppelin. For its special use case, it configures factor where influencer manages and has access on changing parameter of **`Manager`** configs. Only `Enforcer` can propose new parameters.
+
+## Rationale
+
+#### No more proxy of multisigs
+Smart contracts without one's full control is basically a proxy with multisig with smart contract operators. One of members can easily run away with the operating funds as 0xSifu from Magic Internet Money(MIM).
+
+#### Gas saving for end user
+Operating gas cost with one defi contract vs multiple contract was actually gas saving if clone factory pattern was applied. One defi contract storing m users' state globally means each user is actually paying for storage cost on other previous users after interacting with the contract. This makes MakerDAO's contract operating cost to be sometimes over 0.1 ETH, and sometimes limit user's minimum deposit for CDP to save gas cost. To solve inefficient n-times charging gas cost interaction for future users, one contract per user is needed.
+
+#### Separation between investor's and operation fund
+Separation between investor's fund and operation fee is clearly specified in the smart contract, so investors can ensure safety from arbitrary loss of funds by the operating team's control.
+
+
+## Backwards Compatibility
+This EIP has no known backward compatibility issues.
+
+## Reference Implementation
+
+[Reference implementation](../assets/eip-5252) is a simple deposit account contract as `Finance` contract and its contribution value α is measured with deposit amount with ETH.
+
+## Security Considerations
+
+- **`Factory`** contract must ensure that each **`Finance`** contract is registered in the factory and check **`Finance`** contract is sending transaction related to its bounded owner.
+
+- Reentrancy attack guard should be applied or change state before delegatecall in each user function in **`Manager`** contract or **`Finance`** contract. Otherwise, **`Finance`** can be generated as double and ruin whole indices. 
+
+- Once a user locks influence on a proposal's vote, **`Account Bound Token`** cannot be transferred to other wallet. Otherwise, double influence can happen. 
```

Comment:
```suggestion
- Once a user locks influence on a proposal's vote, an **`Account Bound Token`** cannot be transferred to another wallet. Otherwise, double influence can happen. 
```
Required: Grammar